dataflow/gpu-workers/Dockerfile

```diff
@@ -12,22 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM tensorflow/tensorflow:2.4.0-gpu
+# Make sure the CUDA and cuDNN versions are compatible with your TensorFlow version.
```

Comment:
I recommend you check a user journey for user who has old version of Tensorflow, say, 2.1.0, and see if they can figure the configuration given the instructions hear, and whether it works.
I noticed that we often focus in instructions/examples on 'latest' versions, but customers are often several releases back, and there are quirks that they need to do because they are on the old version. So for old versions, we either need to document quirks, or mention to customers: this approach won't work unless you use version x or higher.

Reply:
I tried updating the TensorFlow version to 2.1.0, which would require CUDA 10.1 and cuDNN 7. I manually updated the base image to `nvcr.io/nvidia/cuda:10.1-cudnn7-runtime-ubuntu18.04`, and had to downgrade to Python 3.7 to install the older TensorFlow version. After that, it still worked out of the box.

I think it's straightforward enough to not have to include explicit instructions, the Dockerfile already has comments that the TensorFlow, CUDA and cuDNN versions have to match a compatible configuration, and have a link to the TensorFlow page with the compatible configurations. I added a link to the Nvidia container registry so they can find the image they need.